src/Windows/Core/Impl/Microsoft.Rtvs.Core.Windows.csproj

```diff
@@ -67,7 +67,7 @@
     <Compile Include="Shell\CoreShell.cs" />
     <Compile Include="Shell\CoreShellExtensions.cs" />
     <Compile Include="Shell\ICompositionCatalog.cs" />
-    <Compile Include="Shell\ICoreShell.cs" />
```

Comment:
Probably still need, in minimal form. I think we mostly reduced it to what needed in x-plat: locale, app name, idle, start app and terminate app events.